pkgs/development/python-modules/pyarrow/default.nix

```diff
@@ -25,7 +25,10 @@ buildPythonPackage rec {
     "-DCMAKE_POLICY_DEFAULT_CMP0025=NEW"
   ];
 
+  # Cmake configure step will put us into the build dir
```

Comment:
see the comment about the cmake hook. Do we need to actually run it? I fixed several cases by passing `dontUseCmakeConfigure`.

Reply:
Yea, actually, that works